ts/utils/__tests__/login.test.ts

```diff
@@ -69,3 +70,35 @@ describe("hook the login outcome from the url", () => {
     }
   }
 });
+
+describe("getIntentFallbackUrl", () => {
+  const isIntentSchemeCases: ReadonlyArray<[string, Option<string>]> = [
+    ["", none],
+    ["https://www.google.com", none],
+    ["intent:", none],
+    ["intent://", none],
+    [
+      "intent://domain.test.it/?tranId=abc#Intent;scheme=https;package=com.test.it;S.browser_fallback_url=https://domain.it/?tranId=acb;end",
```

Comment:
No sure whether is intentional, but `intent:/domain.test.it/?tranId=abc#Intent;scheme=https;package=com.test.it;S.browser_fallback_url=https://domain.it/?tranId=acb;end"` will result in a valid URL even though there is only one slash after the protocol.
It may be worth to check against it, there is a `slashes: boolean` property from `URLParse` that is already populated by the parser and should be useful.

Reply:
<img width="944" alt="Screenshot 2021-08-02 at 12 38 41" src="https://user-images.githubusercontent.com/2094604/127849060-84aadb90-001c-4cfb-8b1f-54bcabff34b8.png">

Test added by me, expecting it to fail.

Reply:
Good catch!
Adding it is definitely better!
Add in f53fd72dd2db85a2dd27e0ea41e0fb3f440f16d2